File: src/main/java/no/nav/jfr/generell/operations/eventenricher/saf/SafResponse.java
package no.nav.jfr.generell.operations.eventenricher.saf;

import no.nav.jfr.generell.operations.eventenricher.journalpost.Journalpost;

import java.io.Serializable;

public class SafResponse implements Serializable {
    private Data data;

    public SafResponse() {}

    public Data getData() { return data; }

    public void setData(final Data data) {
        this.data = data;
    }

    public class Data implements Serializable {
        private Journalpost journalpost;

        public Data() {}

        public Journalpost getJournalpost() {
            return journalpost;
        }

        public void setJournalpost(final Journalpost journalpost) {
            this.journalpost = journalpost;
        }
    }
}

File: src/main/kotlin/no/nav/helse/flex/operations/eventenricher/saf/SafResponse.kt
package no.nav.helse.flex.operations.eventenricher.saf

import no.nav.helse.flex.operations.eventenricher.journalpost.Journalpost
import java.io.Serializable

class SafResponse(
    val data: Data
) : Serializable {

    inner class Data(
        var journalpost: Journalpost
    ) : Serializable
}
